product_docs/docs/pgpool/4/installing_extensions/index.mdx

```diff
@@ -1,33 +1,56 @@
 ---
-title: "Installing and managing Pgpool-II extensions"
-navTitle: "Installing extensions"
+navTitle: Installing
```

Comment:
Same comment as I made for PEM. For the navTitle, we should differentiate between installing extensions and installing pgPool. You can see how the old version was "Installing extensions". Right now, the nav tree just shows two "Installing" entries.